website/docs/r/event_trigger.html.markdown

```diff
@@ -0,0 +1,138 @@
+  config_service_id = "SERVICE ID"
+  config_match = <<-EOF
+{
+  "updateDescription.updatedFields": {
+    "status": "blocked"
+  }
+}
+EOF
+  config_project = "{\"updateDescription.updatedFields\":{\"status\":\"blocked\"}}"
+  config_full_document = false
+  config_full_document_before = false
+  event_processors {
+    aws_eventbridge {
+      config_account_id = "AWS ACCOUNT ID"
+      config_region = "AWS REGIOn"
+    }
+  }
+}
+```
+
+### Example Usage: DATABASE with eventbridge
+```hcl
+resource "mongodbatlas_event_trigger" "test" {
+  project_id = "PROJECT ID"
+  app_id = "APPLICATION ID"
+  name = "NAME OF THE TRIGGER"
+  type = "DATABASE"
+  disabled = false
+  config_operation_types = ["INSERT", "UPDATE"]
+  config_operation_type = "LOGIN"
+  config_providers = ["anon-user"]
+  config_database = "DATABASE NAME"
+  config_collection = "COLLECTION NAME"
+  config_service_id = "1"
+  config_match = "{\"updateDescription.updatedFields\":{\"status\":\"blocked\"}}"
+  config_project = "{\"updateDescription.updatedFields\":{\"status\":\"blocked\"}}"
+  config_full_document = false
+  config_full_document_before = false
+  config_schedule = "*"
+  event_processors {
+    aws_eventbridge {
+      config_account_id = "AWS ACCOUNT ID"
+      config_region = "AWS REGIOn"
+    }
+  }
+}
+```
+
+### Example Usage: AUTHENTICATION
+```hcl
+resource "mongodbatlas_event_trigger" "test" {
+  project_id = "PROJECT ID"
+  app_id = "APPLICATION ID"
+  name = "NAME OF THE TRIGGER"
+  type = "AUTHENTICATION"
+  function_id = "1"
+  disabled = false
+  config_operation_type = "LOGIN"
+  config_providers = ["anon-user"]
+}
+```
+
+### Example Usage: SCHEDULED with eventbridge
+```hcl
+resource "mongodbatlas_event_trigger" "test" {
+  project_id = "PROJECT ID"
+  app_id = "APPLICATION ID"
+  name = "NAME OF THE TRIGGER"
+  type = "SCHEDULED"
+  function_id = "1"
+  disabled = false
+  config_schedule = "*"
+}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) The unique ID for the project to create the trigger.
+* `app_id` - (Required) The ObjectID of your application.
+* `name` - (Required) The name of the trigger.
+* `type` - (Required) The type of the trigger. Possible Values: `DATABASE`, `AUTHENTICATION`
+* `function_id` - (Optional) The ID of the function associated with the trigger.
+* `disabled` - (Optional) Default: `false` If `true`, the trigger is disabled.
+* `config_operation_types` - (Optional) The [database event operation types](https://docs.mongodb.com/realm/triggers/database-triggers/#std-label-database-events) to listen for. This must contain at least one value. Required for `DATABASE` type. Possible Values: `INSERT`, `UPDATE`, `REPLACE`, `DELETE`
+* `config_operation_type` - (Optional) The [authentication operation type](https://docs.mongodb.com/realm/triggers/authentication-triggers/#std-label-authentication-event-operation-types) to listen for. Required for `AUTHENTICATION` type. Possible Values: `LOGIN`, `CREATE`, `DELETE`
```

Comment:
Same on the required, move up (though it is weird to have after the Terraform option/required note but I think people get that hopefully.  Can you do the same for all here with a requirement? Example:

* `config_operation_type` - (Optional) Required for `AUTHENTICATION` type.  The [authentication operation type](https://docs.mongodb.com/realm/triggers/authentication-triggers/#std-label-authentication-event-operation-types) to listen for. Possible Values: `LOGIN`, `CREATE`, `DELETE`